sparrow-py/pysrc/sparrow_py/_timestream.py

```diff
@@ -776,6 +792,83 @@ def lookup(self, key: Union[Timestream, Literal]) -> Timestream:
         """
         return Timestream._call("lookup", key, self)
 
+    def shift_to(self, time: Union[Timestream, datetime]) -> Timestream:
+        """
+        Create a Timestream shifting the time of each point to `time`.
```

Comment:
Maybe "shifting each point forward to `time`" (conceputally, shifts the point, not the time of the point)